source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -1217,6 +1222,22 @@ Example:
  <windows_registry arch="64bit" recursion_level="3">HKEY_LOCAL_MACHINE\SYSTEM\Setup</windows_registry>
 
 
+.. versionadded:: 4.6.0
+
+If there is a conflict between a block with wildcards and another without them, the block without wildcards will be used for the specific case. As an example:
+
+.. code-block:: xml
+
+  <windows_registry arch="both" check_sum="no">HKEY_LOCAL_MACHINE\SOFTWARE\*</windows_registry>
+
+The above block will set the ``SOFTWARE`` key of all users to be monitored in scheduled mode.
+
+.. code-block:: xml
+  
+  <windows_registry arch="both" check_sum="no">HKEY_LOCAL_MACHINE\SOFTWARE\TEST_KEY</windows_registry>
```

Comment:
I think `check_sum="yes"` works best for this second key setting. Every key (`*`) found under `SOFTWARE\` won't be scanned for checksum changes (`check_sum="no"`) except for `TEST_KEY` (`check_sum="yes"`) because gets prevalence

Reply:
Done. Thanks Javi !